Bendy and the Dark Revival/bendyandthedarkrevival.asl

```diff
@@ -19,8 +19,12 @@ startup
         { "csc_10201", new List<string>() { "ch_intro" } },
         { "csc_11008", new List<string>() { "ch_1" } },
         { "CHAPTER THREE:", new List<string>() { "ch_2" } },
+        { "csp_11503", new List<string>() { "kw" } },
+        { "Amok Follower: Hail the New Amok!", new List<string>() { "la" } },
```

Comment:
You don't need this either, you don't reference a setting named "Amok Follower: Hail the New Amok!"